sdcm/tester.py

```diff
@@ -1463,60 +1477,18 @@ def tearDown(self):
             self.log.exception('Exception in finalize_test method {}'.format(details))
             raise
         finally:
-            if self._failure_post_behavior == 'destroy':
-                clean_cloud_instances({"TestId": str(cluster.Setup.test_id())})
             stop_events_device()
-            self.collect_events_log()
-            self.zip_and_upload_job_log()
-            stop_rsyslog()
-
-    def zip_and_upload_job_log(self):
+            if self.params.get('collect_logs', False):
```

Comment:
remove the default, it should be set in defaults yaml

Reply:
done